src/vm/analysis/type_checker/natives/mod.rs

```diff
@@ -210,20 +209,24 @@ fn check_special_if(checker: &mut TypeChecker, args: &[SymbolicExpression], cont
 
 fn check_contract_call(checker: &mut TypeChecker, args: &[SymbolicExpression], context: &TypingContext) -> TypeResult {
     check_arguments_at_least(2, args)?;
-    let contract_name = args[0].match_atom()
-        .ok_or(CheckError::new(CheckErrors::ContractCallExpectName))?;
+    let contract_identifier = match args[0].expr {
```

Comment:
And here.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1105/commits/c871057b86ad7ab2c64a84ba2986f42b469860ec